### ISSUE TITLE
unsigned long? lineNumber and columnNumber in FeaturePolicyViolationReportBody

### ISSUE THREAD

**Issue by bakulf** [WICG/feature-policy#224]
Spec says that lineNumber and columnNumber in  FeaturePolicyViolationReportBody interface are "long?". They should be "unsigned long?" to be consistent with DeprecationReportBody and InterventionReportBody from Reporting API.

**Comment by annevk** [comment]
And they should be called lineno and colno to be consistent with ErrorEvent...

See also https://github.com/w3c/webappsec-csp/issues/268.

**Comment by clelland** [comment]
Those fields are still called `lineNumber` and `columnNumber` in https://w3c.github.io/webappsec-csp/#violation-events -- I think we should ensure that the reporting API report bodies are more-or-less consistent with each other, as opposed to making one of them different to match an HTML interface.

(No problem changing it, I'd just want to be sure that people are on board with changing Deprecation, Intervention and CSP Violation events as well)

**Comment by annevk** [comment]
cc @andypaicu

**Comment by andypaicu** [comment]
Yes when it comes to CSP the names should be `lineno` and `colno`

**Comment by clelland** [comment]
Also cc @dcreager @paulmeyer90 for the reporting API

**Comment by paulmeyer90** [comment]
I also think it makes more sense for the names to be consistent with the other report types, and both deprecation and intervention reports are already shipped. I'd be in favor of keeping them the same.
